ansible/roles/ocp-workload-aiedge-computing-blueprint/tasks/workload.yml

```diff
@@ -569,6 +571,18 @@
   command:
     cmd: "oc apply -f '{{ hub_repo_dir }}/base/03_services/argocd-operator-subs/subscription-labels.yaml'"
 
+- name: Create the ACM namespace for the blueprint-management-hub
+  command:
+    cmd: "oc apply -f '{{ hub_repo_dir }}/base/03_services/manuela-factory-subs/namespace.yaml'"
```

Comment:
Any reason we can't use the `k8s` module here?

Reply:
All of these files are cloned from github, modified locally and `git push` to a gitea repo as part of the demo workflow. 
Ideally, I would prefer to use k8s but, afaik, the `file` `lookup` plugin cannot reference files outside of the playbook directory.

Reply:
I'm pretty sure you can use `src` to point to a YAML file on the remote node... No lookup needed unless the file comes from the controller node.

From the k8s module doc:
Provide a path to a file containing a valid YAML definition of an object or objects to be created or updated. Mutually exclusive with resource_definition. NOTE: kind, api_version, name, and namespace will be overwritten by corresponding values found in the configuration read in from the src file.
Reads from the local file system. To read from the Ansible controller's file system, including vaulted files, use the file lookup plugin or template lookup plugin, combined with the from_yaml filter, and pass the result to resource_definition. See Examples below.
Mutually exclusive with template in case of k8s module.

Reply:
I was blinded by using `lookup` that I didn't even think to check and see if `k8s` module added support for reading from a outside path.  If this works then I will definitely update all the `shell` modules that are just running `oc apply`

Reply:
Updated to use the `k8s` module with `src` parameter